### DISCUSSION TITLE
Unsupported Chain

### DISCUSSION THREAD

**Discussion by LilFatFrank** [wevm/wagmi#2077] [Q&A]
When you switch to a chain that is not supported, wagmi throws an error like so

<img width="329" alt="Screenshot 2023-03-27 at 4 23 49 PM" src="https://user-images.githubusercontent.com/30598417/227922193-5d23888d-6609-4379-93a1-c8248d8f46bf.png">

Is there a way to remove this modal?

**Comment by tmm** [comment]
Is this ConnectKit? Better to ask over there as wagmi doesn't control their interface.
